docs/platform/10_known-issues.md

```diff
@@ -9,6 +9,7 @@ sidebar_label: Known Issues
 
 ### [BYOC](https://docs.grid.ai/platform/custom-cloud-credentials/byoc)
 * Creating runs with the same name across two cluster is currenly supported, but prohibits certain operations against runs/experiments.
+* BYOC can only support instance types with the x86_64 architecture.
```

Comment:
@oojo12 can we be any more explicit here? we don't need to name every instance that is supported, but can we link them to AWS docs of x86 architecture or something similar?

Reply:
Restricting a producer can not be correct (e.g., there are also AMD-produced x64 CPU-based instances etc.)
I'd rather make a note the instance families of the following types are not supported due to their architecture: A1, T4g, M6g, C6g, and R6g. Their processors use the `aarch64` (ARM64) architecture instead of `x86_64` (AMD64).  This means that EC2 instances and AMIs (Amazon Machine Images) created using the `x86_64` architecture aren't compatible with instances use the `ARM64` architecture.

Reply:
@essiequoi thoughts before I make this change?

Reply:
I agree. Let's be more explicit than less. I would update this sentence to say something more like: 

BYOC can only support instance types with the x86_64 architecture. The instance families of the following types are not supported due to their architecture: A1, T4g, M6g, C6g, and R6g. 

@oojo12